core/modules/layout/includes/layout.class.inc

```diff
@@ -769,6 +774,10 @@ class Layout {
    * return LayoutContext[]
    */
   function getContexts() {
+    if (is_null($this->contexts)) {
```

Comment:
Same as previous comment:

```suggestion
    if (!is_array($this->contexts)) {
```

Reply:
I've committed these changes, but I'm surprized the `is_null()` check didn't work for you @klonos. What was `$this->contexts` on your site? Was it an empty string? `''`

Reply:
Hmm 🤔 ...I still see `is_null()`, but anyway.

I've made these suggestions before actually testing. I simply went by what was expected by the offending line in the errors (`$this->contexts += $contexts;`) ...so if `$this->contexts` was expected to be an array, I thought that checking if it is not an array would suffice. Besides, `NULL` is not an array, and neither is `''`, so `!is_array()` would've been true in those cases too.

Reply:
...I should have said that when I suggested the `!is_array()` I was coming directly from https://github.com/backdrop/backdrop-issues/issues/4611#issuecomment-689549746